Source/ui_qt/DebugSupport/FunctionsView.cpp

```diff
@@ -0,0 +1,274 @@
+#include <QHeaderView>
+#include <QGridLayout>
+#include <QInputDialog>
+#include <QMessageBox>
+#include <QPushButton>
+
+#include "FunctionsView.h"
+#include "string_cast.h"
+#include "string_format.h"
+#include "lexical_cast_ex.h"
+
+#define DEFAULT_GROUPID (1)
+#define DEFAULT_GROUPNAME ("Global")
+
+CFunctionsView::CFunctionsView(QMdiArea* parent)
+    : QMdiSubWindow(parent)
+{
+
+	resize(300, 700);
+
+	parent->addSubWindow(this);
+	setWindowTitle("Functions");
+
+	m_treeWidget = new QTreeWidget(this);
```

Comment:
Could we convert the UI creation code to a Qt UI file?

Reply:
for all of them or just this view?

Reply:
All of them if possible.